app/styles/_base.scss

```diff
@@ -1,3 +1,5 @@
+@import 'vars';
+
```

Comment:
A couple of points more:

1. Overlapping on smaller screens: 
<img width="506" alt="screen shot 2016-12-31 at 2 37 24 pm" src="https://cloud.githubusercontent.com/assets/7583815/21576865/f9bfbfae-cf66-11e6-953f-4e88b087882c.png">

2. The box appears to be a bit misaligned due to the addition of shadow: 
<img width="459" alt="screen shot 2016-12-31 at 2 39 56 pm" src="https://cloud.githubusercontent.com/assets/7583815/21576867/3aeeabb6-cf67-11e6-9b5d-88fd8fd9140b.png">